### PR TITLE
getObjective method implemented.

### PR DESCRIPTION
**Pull request by Vgrunert**

### CODE REVIEW

File: src/JuMP.jl [hunk 1]
```diff
@@ -199,6 +199,8 @@ end
 @Base.deprecate getNumVars(m::Model) MathProgBase.numvar(m)
 @Base.deprecate getNumConstraints(m::Model) MathProgBase.numlinconstr(m)
 
+getCost(m::Model) = m.redCosts
```
Comment (mlubin) [inline src/JuMP.jl]: This is already available through ``getDual(::Variable)``

Comment (Vgrunert) [inline src/JuMP.jl]: Not exactly. The getDual returns the costs of one variable, whereas the redCosts returns the costs associated with the entire model.

**Comment by mlubin** [comment]
Could you describe a bit what you're trying to do and why these new methods are needed?

File: src/JuMP.jl (hunk 1 above)
Comment (mlubin) [inline src/JuMP.jl]:
```
n = MathProgBase.numvar(model)
z = zeros(n)
for i in 1:n
    z[i] = getDual(Variable(model,i))
end
```
Accomplishes this using the existing documented methods.

**Comment by Vgrunert** [comment]
Yes. The getObjective function was added simply because it was not implemented so far. I tried to use it but could not, so I added it. The getCost function is useful to get an overview of the reduced costs / shadow costs of the entire model, compared to the getDual function which provides the costs of a particular variable.

File: src/JuMP.jl (hunk 1 above)
Comment (Vgrunert) [inline src/JuMP.jl]: Fair enough. You decide of course. I just thought that it's a nice convenience function. But please consider the getObjective function, as it is being exported but not implemented so far.

**Comment by mlubin** [comment]
Yes, exported but not implemented is a bit bizarre, not sure how that happened. @IainNZ

**Comment by Vgrunert** [comment]
Well Thanks for your consideration. I was also thinking about adding a few docstrings in the near future, so until then all the best.

**Comment by mlubin** [comment]
I think it's a good idea to provide ``getObjective``, along with way to access constraints (https://github.com/JuliaOpt/JuMP.jl/issues/462)

**Comment by IainNZ** [comment]
Yes to getObjective, no to getCost, docstrings is good but lets hold off for now until we think about style.

**Comment by Vgrunert** [comment]
Fair enough.

**Comment by joehuchette** [comment]
I think we should merge da361e5829421ce53c015c371601668cd7c35046

**Comment by mlubin** [comment]
Needs docs and tests and handling for nonlinear objectives

**Comment by Vgrunert** [comment]
Sry, I am not familiar with github. Is there anything on my part that needs to be done? I though that the request is already out and has to be accepted.

**Comment by mlubin** [comment]
You can make changes to your PR at any time by modifying the corresponding branch on your fork (in this case ``master``). If you're up to it you could tidy the PR by removing the ``getCost`` changes and adding documentation for ``getObjective``, otherwise we'll do it when we get a chance.

**Pull request closed without merging**